nats/src/lib.rs

```diff
@@ -14,7 +14,7 @@ extern crate log;
 
 use codec::capabilities::{CapabilityProvider, Dispatcher, NullDispatcher};
 
-pub const OP_DELIVER_MESSAGE: &str = "DeliverMessage";
+pub const OP_HANDLE_MESSAGE: &str = "HandleMessage";
```

Comment:
We should be able to get these constants from the wasmcloud-messaging-provider crate, which needs a version bump to correspond to the breaking API change.